### PR TITLE
Add aro-e2e workflow

### PR DESCRIPTION
**Pull request by slintes**
Needs access to RH network, configure test according to https://docs.ci.openshift.org/docs/architecture/ci-operator/#allowing-tests-to-access-red-hat-intranet

Test also needs https://github.com/openshift/ci-tools/pull/4318 and https://github.com/openshift/release/pull/56644

### CODE REVIEW

File: ci-operator/step-registry/aro-e2e/classic/aro-e2e-classic-workflow.yaml
Comment (stbenjam) [inline ci-operator/step-registry/aro-e2e/classic/aro-e2e-classic-workflow.yaml:19]: Is it not possible to use an arbitrary release payload yet?

Comment (slintes) [reply ci-operator/step-registry/aro-e2e/classic/aro-e2e-classic-workflow.yaml:19]: I want to keep it simple in the first iteration, but in a follow up the  plan is to get this from the `releases` config in the actual test.

**Comment by slintes** [comment]
/hold

I'm going to create a test job which uses this, so we can run rehearsals on this PR

**Comment by slintes** [comment]
/hold cancel

I can't create a test job using this workflow without having this workflow merged 🤷🏼‍♂️ 

New plan:
- merge this without knowing if it works
- create and merge a test job (I would be suprised if the workflow works out of the box)
- create a follow PR for this workflow which fixes issues using rehearsal jobs

**Review by tsatam — APPROVED** [review]

**Review by tsatam — APPROVED** [review]

**Pull request merged**
